client/src/components/Form/InputDisplay/InputDisplay.tsx

```diff
@@ -55,14 +70,14 @@ const InputDisplay = (props) => {
                 />)
                 : null
             }
-            <Col xm={12} lg={6}>
+            <Col xs={12} lg={6}>
```

Comment:
I think it should be sm and not xs. Based on the InputDropdown.
xs and sm are both valid.
Before, both files contained 'xm', so change it to sm or xs both. Or tell me the cause of your decision.

Reply:
'xs' is the correct value. Fixed.

Reply:
So xs in both files, or xs here, and spm in InputDropdown?